wordpress-mobile/WordPress-Android
# Quick Start for Existing Users V2 - Changes to existing task - Stats

Fixes #16349 

This PR has the following changes 

#### Focus points on Quick Link Ribbon 
- For the below quick start tasks
   -  1. **Check your site stats**
   -  2. **Edit your Home page**
   -  3. **Review site pages**  

If the user is on dashboard tab, then the focus point is shown on the Quick Links ribbon

#### Stats 
- Updates the task title to `Check your site stats`
- Updates the task description to `Keep up to date with your site\'s performance`
- Updates Notice for Stats to `Select stats to see how your site is performing`

---

### To test:
Test 1 
- Launch the app with a site having a quick start in progress 
- Go to My Site -> Me -> App Settings -> Debug settings.
- Turn off the MySiteDashboardTabsFeatureConfig flag under the Features in development section.
Restart the app.
- Logout/login and select "Show me around" when the quick start prompt is shown.
- Run tests in [wpandroid-quick-start-testing-instructions.txt](https://github.com/wordpress-mobile/WordPress-Android/files/8246096/wpandroid-quick-start-testing-instructions.txt).
- Notice that for the following tasks - **Edit your home Page**, **Review Pages** and **Check stats**, the QuickStart focus point is shown in Site Menu Items 
- Notice that for the **Check stats** quick start task, the task title, description and Notice are updated as described in the PR

Test 2 & 3 - Prerequisites
- Go to My Site -> Me -> App Settings -> Debug settings.
- Enable `MySiteDashboardTabsFeatureConfig`
- Restart the app
   
Test 2 - When the default tab is `Site Menu` 
- Navigate to `My Site` -> `Me` -> `App Settings` 
- Tap `Initial Screen` and select `Menu` 
- Launch the app with a site having a quick start in progress 
- Run tests in [wpandroid-quick-start-testing-instructions.txt](https://github.com/wordpress-mobile/WordPress-Android/files/8246096/wpandroid-quick-start-testing-instructions.txt).
- Notice that for the following tasks - **Edit your home Page**, **Review Pages** and **Check stats**, the QuickStart focus point is shown in the Site menu list items. 
- Notice that for the quick start tasks  - **Check your Site, Social Sharing and Explore plans**, the Quickstart focus points are shown in the Site menu items as same as before. 
- Notice that for the **Check stats** quick start task, the task title, description and Notice are updated as described in the PR

Test 3 - When the default tab is `Home`
- Navigate to `My Site` -> `Me` -> `App Settings` 
- Tap `Initial Screen` and select `Home` 
- Launch the app with a site having a quick start in progress 
- Run tests in [wpandroid-quick-start-testing-instructions.txt](https://github.com/wordpress-mobile/WordPress-Android/files/8246096/wpandroid-quick-start-testing-instructions.txt).
- Notice that for the following tasks - **Edit your home Page**, **Review Pages** and **Check stats**, the QuickStart focus point is shown in the Quick links Ribbon Card. 
- Notice that for the quick start tasks  - **Check your Site, Social Sharing and Explore plans**, the Quickstart focus points is shown in the Site menu tab header first
- Notice that for the **Check stats** quick start task, the task title, description and Notice are updated as described in the PR
- On clicking the Site menu tab header, the list gets scrolled to show the Site menu item corresponding to the task 



---

### Known Issues
Adding the quick start focus point in the quick link ribbon reintroduces this bug #16288 


## Regression Notes
1. Potential unintended areas of impact
- Quick start focus points not shown properly 
- Stats Quick start task has incorrect title, description and notice 


2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests and Unit tests

3. What automated tests I added (or what prevented me from doing so)
Unit tests 

PR submission checklist:
